tests/suites/cloud/suite.js

```diff
@@ -231,7 +232,22 @@ module.exports = {
     config.deviceId = deviceRegInfo.id;
     config.persistentLogging = true;
     config.developmentMode = true;
-    config.installer = { secureboot: ['1', 'true'].includes(process.env.FLASHER_SECUREBOOT) };
+    config.installer = {
+      secureboot: ['1', 'true'].includes(process.env.FLASHER_SECUREBOOT),
+      migrate: { force: this.suite.options.installerForceMigration }
```

Comment:
I guess this should be `this.suite.options.config.installerForceMigration` as we expect `installerForceMigration` inside the `config` section.

Reply:
@mtoman not really, it's not a direct configuration path, first `config.js` is parsed into the Suite class in https://github.com/balena-os/leviathan/blob/2ac221ae40fd3063335be2fc0d43c2e8b3ca3286/core/lib/common/suite.js#L91, and here `installerForceMigration` is directly under options.

Whether I should move it into the `balenaOS.config` section is a different question - I didn't find the organization there clean cut.

Reply:
@mtoman I will actually move it under `balenaOS.config` on a different PR as it makes more sense. But this PR works as is and should unblock the migrator & secureboot tests.